packages/common.vm/common.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>common.vm</id>
-    <version>0.0.0.20230714</version>
+    <version>0.0.0.20230718</version>
```

Comment:
We need increase this version to today's date as this is now not increasing the version.